chat-client/src/client/mynahUi.ts

```diff
@@ -52,13 +95,53 @@ export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [Myna
             }
         }
 
-        mynahUi.addToUserPrompt(tabId, params.prompt)
+        return tabId
+    }
+
+    const sendToPrompt = (params: SendToPromptParams) => {
+        const tabId = getOrCreateTabId()
+        if (!tabId) return
```

Comment:
Is it expected we can have no `tabId` even after creating new tab?

Reply:
Yes, if we have reached the limit for allowed tabs